src/sql/azdata.proposed.d.ts

```diff
@@ -1111,6 +1111,10 @@ declare module 'azdata' {
 			 * The input validation error.
 			 */
 			inputValidationError?: string;
+			/**
+			 * Properties related to the table to be captured
+			 */
+			propertyBag?: { [key: string]: string };
```

Comment:
let's rename it to telemetryInfo and add proper comments.

Reply:
Why limit this to telemetry? Sure that may be the primary use right now, but it's possible that this would be used for something else in the future. Just `metadata` would be better IMO

Reply:
(also in general we should avoid putting anything about "telemetry" in the public extension API...)